var/spack/repos/builtin/packages/icu4c/package.py

```diff
@@ -12,10 +12,8 @@ class Icu4c(AutotoolsPackage):
     C/C++ interface."""
 
     homepage = "http://site.icu-project.org/"
-    url      = "http://download.icu-project.org/files/icu4c/57.1/icu4c-57_1-src.tgz"
```

Comment:
You’ll still want a top-level URL so commands like `spack versions` and `spack checksum` will work.

Reply:
That was an accidental deletion. Thanks for catching that.